objects/github_tickets.rb

```diff
@@ -99,6 +99,14 @@ def users
     end
   end
 
+  def truncate(text, limit = 40, separator = '...')
+    return text if limit >= text.length
+
```

Comment:
@vivekimsit remove this empty line, see http://www.yegor256.com/2014/11/03/empty-line-code-smell.html